jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
assign to True looks clearer. If you want to use integral, then anything not equal to 0 will do. To your question, both are correct, I prefer the assign to True.